File: Strings/__3000__/3358/3358.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf-8')
const lines = input.split('\n')
const vogais = ['a', 'e', 'i', 'o', 'u']
const n = Number(lines.shift())

for(let i = 0; i < n; i++){
    let is_hardest = false
    let auxiliar = lines.shift()
    let curr = aux.toUpperCase()

    for(let j = 0; j < curr.length; j++){
        let count = 0

        while(
            count < 3 && 
            j + count < curr.length && 
            !vogais.includes(curr[j + count])
        ){ count++ }

        if(count == 3){
            is_hardest = true
            break
        }
    }

    if(is_hardest){
        console.log(`${auxiliar} nao eh facil`)
    } else {
        console.log(`${auxiliar} eh facil`)
    }
}

File: Strings/__3000__/3358/3358.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int main(){

    int n;
    cin >> n;
    vector<char> vogais = {'a', 'e', 'i', 'o', 'u'};

    for(int i = 0; i < n; i++){

        int count = 0;
        bool hard = false;
        string sobrenome;
        cin >> sobrenome;

        for(char letter: sobrenome){
            char aux = tolower(letter);

            if(find(vogais.begin(), vogais.end(), aux) == vogais.end()){
                count++;
            } else {
                count = 0;
            }

            if(count == 3){
                hard = true;
                break;
            }
        }

        if(hard){
            cout << sobrenome << " nao eh facil" << endl;
        } else {
            cout << sobrenome << " eh facil" << endl;
        }

    }

    return 0;
}